apiserver/facades/agent/uniter/networkinfo_test.go

```diff
@@ -0,0 +1,432 @@
+// Copyright 2019 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package uniter_test
```

Comment:
I haven't looked over the tests yet. Or had time to think of the broader context.

Reply:
These are just the tests for `NetworksForRelation` that were relocated from the `state` package.